### ISSUE TITLE
model

### ISSUE THREAD

**Issue by YZYLHR** [AbnerHqC/GaitSet#100]
Hi! The models in the checkpoint/GaitSet is under the expermental condition of 74/50. I want to use the model under 62/62. Does it need retraining?

**Comment by AbnerHqC** [comment]
Positive! Otherwise, your test set will be overlap with the training set.
